### ISSUE TITLE
Failed to generate vector assets

### ISSUE THREAD

**Issue by jagadishg** [oblador/react-native-vector-image#14]
Hi,

I have installed the necessary packages.
`yarn add react-native-vector-image @klarna/react-native-vector-drawable`

I have added the following line to the React Native Build Phase
`../node_modules/react-native-vector-image/strip_svgs.sh`

When I run the `yarn react-native-vector-image generate` command, I get the following error

```
While trying to resolve module `nanoid/non-secure` from file `/Users/someuser/Documents/workspace/shop-mobile/node_modules/@react-navigation/routers/lib/commonjs/BaseRouter.js`, the package `/Users/someuser/Documents/workspace/shop-mobile/node_modules/nanoid/non-secure/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/Users/someuser/Documents/workspace/shop-mobile/node_modules/nanoid/non-secure/index.cjs`. Indeed, none of these files exist:

  * /Users/someuser/Documents/workspace/shop-mobile/node_modules/nanoid/non-secure/index.cjs(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)
  * /Users/someuser/Documents/workspace/shop-mobile/node_modules/nanoid/non-secure/index.cjs/index(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)
Error: While trying to resolve module `nanoid/non-secure` from file `/Users/someuser/Documents/workspace/shop-mobile/node_modules/@react-navigation/routers/lib/commonjs/BaseRouter.js`, the package `/Users/someuser/Documents/workspace/shop-mobile/node_modules/nanoid/non-secure/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/Users/someuser/Documents/workspace/shop-mobile/node_modules/nanoid/non-secure/index.cjs`. Indeed, none of these files exist:

  * /Users/someuser/Documents/workspace/shop-mobile/node_modules/nanoid/non-secure/index.cjs(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)
  * /Users/someuser/Documents/workspace/shop-mobile/node_modules/nanoid/non-secure/index.cjs/index(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)
    at DependencyGraph.resolveDependency (/Users/someuser/Documents/workspace/shop-mobile/node_modules/metro/src/node-haste/DependencyGraph.js:436:17)
    at Object.resolve (/Users/someuser/Documents/workspace/shop-mobile/node_modules/metro/src/lib/transformHelpers.js:317:42)
    at resolve (/Users/someuser/Documents/workspace/shop-mobile/node_modules/metro/src/DeltaBundler/traverseDependencies.js:629:33)
    at /Users/someuser/Documents/workspace/shop-mobile/node_modules/metro/src/DeltaBundler/traverseDependencies.js:645:26
    at Array.reduce (<anonymous>)
    at resolveDependencies (/Users/someuser/Documents/workspace/shop-mobile/node_modules/metro/src/DeltaBundler/traverseDependencies.js:644:33)
    at /Users/someuser/Documents/workspace/shop-mobile/node_modules/metro/src/DeltaBundler/traverseDependencies.js:329:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/someuser/Documents/workspace/shop-mobile/node_modules/metro/src/DeltaBundler/traverseDependencies.js:137:24)
    at _next (/Users/someuser/Documents/workspace/shop-mobile/node_modules/metro/src/DeltaBundler/traverseDependencies.js:159:9)
Failed to generate vector assets
✨  Done in 9.03s.
```
What am I missing here? I'm using react-navigation package for screen navigation.

**Comment by oblador** [comment]
Hi, what version of react-native are you using and which package manager (npm/yarn) + version are you using?

**Comment by jagadishg** [comment]
I'm using the following

yarn: 1.22.4
react: 17.0.1 => 17.0.1
react-native: 0.64.1 => 0.64.1

**Comment by oblador** [comment]
Hmm, it's same as the Example app so should be fine. Can you post the contents of your `metro.config.js`?

**Comment by jagadishg** [comment]
Sure. Here is the metro.config.js

```
module.exports = {
  transformer: {
    getTransformOptions: async () => ({
      transform: {
        experimentalImportSupport: false,
        inlineRequires: true,
      },
    }),
  },
};
```

**Comment by oblador** [comment]
That looks good, and if you run `yarn why metro`?

Does the app run fine by itself otherwise? Any custom setups?

**Comment by oblador** [comment]
Anyways, I'm able to replicate this locally, so I'm fairly sure it's not your setup. Will dig deeper later!

**Comment by jagadishg** [comment]
I was also able to reproduce it with the Example app. The problem appears when I use react-navigation package.

**Comment by oblador** [comment]
Fixed in [0.3.2](https://github.com/oblador/react-native-vector-image/releases/tag/v0.3.2).
